src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -975,7 +975,7 @@ export const ActiveRecordingRow: React.FC<ActiveRecordingRowProps> = ({
         <Td key={`active-table-row-${index}_6`} dataLabel={tableColumns[4].title}>
           <LabelGroup isVertical style={{ padding: '0.2em' }}>
             {recordingOptions(recording).map((options) => (
-              <Label color="blue" key={options.key} isTruncated>
+              <Label color="blue" key={options.key} textMaxWidth="20ch">
```

Comment:
I notice that other places using this attribute use it like `textMaxWidth={'20ch'}`. This simple `"20ch"` seems nicer, just less visual clutter to read. Maybe update the others to use this style?

And also, maybe extracting this to a constant somewhere would be worthwhile.

Reply:
Sounds good! Now, it will look like `textMaxWidth={LABEL_TEXT_MAXWIDTH}`